docs/design/libraries/ComInterfaceGenerator/VTableStubs.md

```diff
@@ -80,26 +80,29 @@ public readonly ref struct VirtualMethodTableInfo
     }
 }
 
-public interface IUnmanagedVirtualMethodTableProvider<T> where T : IEquatable<T>
+public interface IUnmanagedVirtualMethodTableProvider
 {
-    protected VirtualMethodTableInfo GetVirtualMethodTableInfoForKey(T typeKey);
+    protected VirtualMethodTableInfo GetVirtualMethodTableInfoForKey(Type type);
 
-    public sealed VirtualMethodTableInfo GetVirtualMethodTableInfoForKey<TUnmanagedInterfaceType>()
-        where TUnmanagedInterfaceType : IUnmanagedInterfaceType<T>
+    public sealed VirtualMethodTableInfo GetVirtualMethodTableInfoForKey()
```

Comment:
Did you mean to remove this type parameter? It is still referenced in the method body.

```suggestion
    public sealed VirtualMethodTableInfo GetVirtualMethodTableInfoForKey<TUnmanagedInterfaceType>()
```

There are other differences between this documentation and the interfaces in `src/libraries/System.Runtime.InteropServices/tests/Ancillary.Interop/IUnmanagedVirtualMethodTableProvider.cs`, like `IUnmanagedInterfaceType` still taking a type parameter, but I'm less sure if that is intentional or or not.